### DISCUSSION TITLE
Miniplayer

### DISCUSSION THREAD

**Discussion by Rosedog09** [organicmaps/organicmaps#6567] [Ideas]
When home button is pressed display navigation in small overlay above other apps. 
![Screenshot_20230622-125551~2](https://github.com/organicmaps/organicmaps/assets/86130469/19296cf5-ea05-430a-8162-ce3dfb27b288)

**Comment by Jean-BaptisteC** [comment]
Please check similar issues

**Comment by Rosedog09** [comment]
> Please check similar issues

I don't see any similar issues. Im new to Github, but when I search miniplayer, picture in picture, or navigation overlay the closest issue to mine relates to android auto. Am I missing something?

**Comment by Handrail9** [comment]
> Please check similar issues

I would also like to know if there's a different issue to give thumbs up too, I don't see one on the issues page.
